docker-compose.yaml

```
@@ -0,0 +1,12 @@
+version: '3.8'
+
+services:
+  web:
+    command: python3 -m flask run -h 0.0.0.0 -p 8000 --debugger --reload
```

There's not point in specifying `python3` when it's already based on the image

```suggestion
    command: python -m flask run -h 0.0.0.0 -p 8000 --debugger --reload
```